scripts/yul_coverage.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
```suggestion
#!/usr/bin/env bash -e
```
+1 for using `-e` :)

Reply:
I just checked this. Looks like that it can lead to problems on some OS. See https://github.com/koalaman/shellcheck/wiki/SC2096

Reply:
ok. Good point.